recipe/meta.yaml

```
@@ -84,6 +84,7 @@ source:
 
 build:
   number: {{ build_number }}
+  skip: true  # [win and build_type == "debug"]
```

I haven't yet looked into what caused the rendering issue with `conda-smithy`, but maybe we can try
```suggestion
  skip: true  # [win and build_type != "release"]
```
instead.

---

Oh, I just noticed that the last rerendering messed up the non-Windows builds, too. Need to investigate...

---

Yeah this PR has become a mess. I need to clean it up a bit.